docs/howto/outside_controller_use.md

```diff
@@ -0,0 +1,43 @@
+## Using AMS Outside Of A Controller
+
+### Serializing a resource 
+
+In the current version of AMS, serializing resources outside controllers is
+fairly simple:
+
+```ruby
+# Create our resource
+post = Post.create(title: "Sample post", body: "I love Active Model Serializers!")
```

Comment:
TODO (in future PRs):

- [ ] example resource doesn't need to be persisted or even an ActiveRecord object
  - [ ] unless your unpersisted resources needs an attribute only available in the db
  - [ ] unless your 'poro' needs complicated associations
- [ ] Show [ActiveModel::Serializer::Lint::Tests](https://github.com/rails-api/active_model_serializers/blob/1388ae82f26133d62f4e6cdfc40d1c659e121935/lib/active_model/serializer/lint.rb) and [usage](https://github.com/rails-api/active_model_serializers/blob/1388ae82f26133d62f4e6cdfc40d1c659e121935/test/poro_test.rb)